public/style.css

```diff
@@ -219,3 +219,169 @@ li:hover {
 }
 
 /*Welcome Page end*/
+/* RESULT PAGE UPTO -------- LINE-261 */
+.result {
+  width: 100vw;
+  height: 100vh;
+  background-image: url(https://wallpapercave.com/wp/wp8862729.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  color: white;
+  font-family: 'Designer Notes Bold', sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 7rem
+}
+
+.score {
+  object-fit: cover;
+  width: 200px;
+  height: 200px;
+  position: absolute;
+  top: 10px;
+  left: 40%;
+  transition: all 1s ease;
+  border-radius: 50%;
+  ;
```

Comment:
This line does nothing

Reply:
yeah..didnt see that one...
fixed......